ecs-cli/modules/utils/utils.go

```diff
@@ -106,3 +107,40 @@ func GetPartition(region string) string {
 		return "aws"
 	}
 }
+
+// ParseLoadBalancers returns a list of load balacners struct
+// For users wanting to register multiple or one load balancers to a single service
+func ParseLoadBalancers(flagValue []string, list []*ecs.LoadBalancer) ([]*ecs.LoadBalancer, error) {
+
```

Comment:
Same as comment above, you can actually declare an empty `[]*ecs.LoadBalancer` here then return this variable.